sources/osgStats/Buffer.js

```diff
@@ -0,0 +1,76 @@
+var MACROUTILS = require('osg/Utils');
+var BufferArray = require('osg/BufferArray');
+var Geometry = require('osg/Geometry');
+var primitiveSet = require('osg/primitiveSet');
+var DrawElements = require('osg/DrawElements');
+
+var Buffer = function(nbCharacters, usageHint) {
```

Comment:
Maybe better to rename it as StatsBuffer to not collide with other possible "Buffer" in user/osgjs code.